todo/src/reducers/index.js

```diff
@@ -0,0 +1,39 @@
+import { ADDITEM, STRIKEITEM } from "../actions";
+
+const initialState = {
+  todos: [
+    {
+      value: "default item",
+      completed: false,
+      id: 0
+    }
+  ],
+  user: [
+    {
+      name: "mike"
+    }
+  ]
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case ADDITEM:
+      let newArr = state.todos.slice();
```

Comment:
Arrays and objects can likely be assigned to `const` variables instead of `let`. This will still keep the `type` the same while still allowing the contents `inside` the array or object to change.